tests/yadisk_test.py

```diff
@@ -307,3 +307,26 @@ async def test_get_operation_status_request_url(self):
             "https://asd8iaysd89asdgiu")
         self.assertTrue(is_operation_link(request.url))
         self.assertTrue(request.url.startswith("https://"))
+
+    @async_test
+    async def test_is_file(self):
+
```

Comment:
Reference the issue/PR associated with the added test

```suggestion
        # See https://github.com/ivknv/yadisk-async/pull/6

```